lib/inc/drogon/HttpResponse.h

```diff
@@ -96,6 +96,18 @@ class DROGON_EXPORT ResponseStream
         return asyncStream_->send(oss.str());
     }
 
+    bool send(const char *data, size_t len)
+    {
+        if (!asyncStream_)
+        {
+            return false;
+        }
+        std::ostringstream oss;
+        oss << std::hex << len << "\r\n";
+        oss << std::string_view{data, len} << "\r\n";
```

Comment:
why are you "dictating" that the format for binary should be `<length><binary data>`?
How many bytes does `len` occupy?
Why are you using a `string_view` for this purpose and not a `span` which would be better suited.

Reply:
there is no std::span in cpp17